src/algorithms/linear_stretching.jl

```diff
@@ -2,13 +2,17 @@
 """
 ```
     LinearStretching <: AbstractHistogramAdjustmentAlgorithm
-    LinearStretching(; minval = 0, maxval = 1)
+    LinearStretching(; [src_minval], [src_maxval], dest_minval = 0, dest_maxval = 1)
+
+    LinearStretching((src_minval, src_maxval) => (dest_minval, dest_maxval))
+    LinearStretching((src_minval, src_maxval))
+    LinearStretching(nothing => (dest_minval, dest_maxval))
 
     adjust_histogram([T,] img, f::LinearStretching)
     adjust_histogram!([out,] img, f::LinearStretching)
 ```
 
-Returns an image where the range of the intensities spans the interval [`minval`, `maxval`].
+Returns an image where the range of the intensities spans the interval [`dest_minval`, `dest_maxval`].
```

Comment:
I suggest we use `dst_minval` and `dst_maxval` instead of `dest_minval` and `dest_maxval` everywhere. The abbreviation `src` is based on removing the vowels from `source` and taking the first three remaining letters. Hence, it feels more appropriate to keep the three-letter abbreviation `dst` for the sake of symmetry.

Reply:
changed in a703029